org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -128,3 +128,10 @@ function bootstrapcivicrm_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
 function bootstrapcivicrm_civicrm_pageRun($page) {
   CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
 }
+
+/**
+ * This hook is invoked after all the content of a CiviCRM form or page is generated. It allows for direct manipulation of the generated content.
+ */
+function bootstrapcivicrm_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
+  $content = '<script src="'.CRM_Extension_System::singleton()->getMapper()->keyToUrl('org.civicrm.bootstrapcivicrm').'/js/radio-checkbox.js"></script>'. $content;
```

Comment:
Ok then you can simply inject the script both in _pageRun and in _buildForm, to have some consistency with the type of hook used